AUTHORS

```diff
@@ -19,6 +19,7 @@ Google Inc.
 JianXiong Zhou <zhoujianxiong2@gmail.com>
 Lei Xu <eddyxu@gmail.com>
 Matt Clarkson <mattyclarkson@gmail.com>
+Nick Hutchinson <nshutchinson@gmail.com>
```

Comment:
This doesn't really matter, but what is the difference between "authors" and "contributors"?